common/network/components/receiver.c

```diff
@@ -9,11 +9,14 @@
 #include "ciri/node.h"
 #include "common/logger_helper.h"
 
+static char const receiver_component_logger[] = "receiver_component";
```

Comment:
Any reason to prefer that over #define, there are already some components (storage) with defines, we should align to the same convention

Reply:
Generally I tend to use static const vars instead of #define as they provide type-safety and debugging symbols. But there is a memory overhead so maybe in this context #define are perfectly fine.